app/src/main/java/vn/sunasterisk/music_70/service/MediaService.kt

```diff
@@ -126,6 +126,24 @@ class MediaService : Service(), HandlerListenerPlayMusic {
         return listTrack[random.nextInt(listTrack.size)]
     }
 
+    fun getCurrentTrack() = currentTrack
+
+    fun setShuffleType(shuffle: Int) {
+        managerPlayingMusic.shuffleType = shuffle
+    }
+
+    fun getShuffleType(): Int = managerPlayingMusic.shuffleType
```

Comment:
```suggestion
    fun getShuffleType() = managerPlayingMusic.shuffleType
```
Thanh niên đẩy code tiết kiệm thế.